PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+&lt;Title&gt; : &lt;One line description&gt;
+&lt;Title can be a one or two words to describe the subsystem or the aspect
+ this pr applies to. Example: Build, Docs, HTTP, Upstream, SSl, ...&gt;
+
+
+*Description:* &lt;What does this PR do? What was the behavior before the PR?
+What is the behavior with the PR? If fixing a bug, please describe what
+the original issue is and how the change resolves it. How does this
+feature get enabled? By default, config change, etc...&gt;
+
+[Optional *Fixes:* # issue]
+[Optional *API Changes:* Link to [Data Plane PR](https://github.com/envoyproxy/data-plane-api/pulls)]
+
+*Risk Level:* Low | Medium | High
```

Comment:
Did we decide if we're going to have some bot to ensure that the tag and values are well formed here?

Reply:
We should open an issue for a bot that can do stuff like this, but I don't see any obvious resources for that right now so I don't think we should block this on that?

Reply:
Yeah, this shouldn't block this PR, but I think it would be good to add this. If someone can create/provide a Python bot template, I can easily hack this up as well, I just don't want to waste time futzing with the GH integration bits.